examples/jetpack_compose/WORKSPACE

```diff
@@ -9,138 +19,82 @@ archive_repository(
+    artifacts = [
+        # Workaround to add missing 'sun.misc' dependencies to 'kotlinx-coroutines-core-jvm' artifact
+        # Check root BUILD file and 'override_targets' arg of a primary 'maven_install'
+        "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.5.1",
     ],
+    fetch_sources = True,
+    repositories = ["https://repo1.maven.org/maven2"],
 )
 
-load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
-
-stardoc_repositories()
-
-## Import Skylib
-
-_SKYLIB_VERSION = "1.0.2"
-
-_SKYLIB_SHA = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44"
-
 http_archive(
     name = "bazel_skylib",
-    sha256 = _SKYLIB_SHA,
-    urls = [
-        "https://github.com/bazelbuild/bazel-skylib/releases/download/{0}/bazel-skylib-{0}.tar.gz".format(_SKYLIB_VERSION),
-    ],
-)
-
-load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
-
-bazel_skylib_workspace()
-
-## Protobuf
-
-_PROTOBUF_VERSION = "3.14.0"
-
-_PROTOBUF_SHA = "bf0e5070b4b99240183b29df78155eee335885e53a8af8683964579c214ad301"
-
-http_archive(
-    name = "com_google_protobuf",
-    sha256 = _PROTOBUF_SHA,
-    strip_prefix = "protobuf-{}".format(_PROTOBUF_VERSION),
-    urls = [
-        "https://github.com/protocolbuffers/protobuf/archive/v{}.zip".format(_PROTOBUF_VERSION),
-    ],
-)
-
-load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
-
-protobuf_deps()
-
-## Rules PKG
-
-_RULES_PKG_VERSION = "0.2.4"
-
-_RULES_PKG_SHA = "4ba8f4ab0ff85f2484287ab06c0d871dcb31cc54d439457d28fd4ae14b18450a"
-
-http_archive(
-    name = "rules_pkg",
-    sha256 = _RULES_PKG_SHA,
-    urls = [
-        "https://github.com/bazelbuild/rules_pkg/releases/download/{0}/rules_pkg-{0}.tar.gz".format(_RULES_PKG_VERSION),
-    ],
+    sha256 = versions.SKYLIB_SHA,
+    urls = ["https://github.com/bazelbuild/bazel-skylib/releases/download/%s/bazel-skylib-%s.tar.gz" % (
+        versions.SKYLIB_VERSION,
+        versions.SKYLIB_VERSION,
+    )],
 )
 
 ## Android
 
-_RULES_ANDROID_VERSION = "0.1.1"
-
-_RULES_ANDROID_SHA = "cd06d15dd8bb59926e4d65f9003bfc20f9da4b2519985c27e190cddc8b7a7806"
-
 http_archive(
-    name = "build_bazel_rules_android",
-    sha256 = _RULES_ANDROID_SHA,
-    strip_prefix = "rules_android-{}".format(_RULES_ANDROID_VERSION),
-    urls = [
-        "https://github.com/bazelbuild/rules_android/archive/v{}.zip".format(_RULES_ANDROID_VERSION),
-    ],
+    name = "rules_android",
+    sha256 = versions.ANDROID.SHA,
+    strip_prefix = "rules_android-%s" % versions.ANDROID.VERSION,
+    urls = ["https://github.com/bazelbuild/rules_android/archive/v%s.zip" % versions.ANDROID.VERSION],
 )
 
-load("@build_bazel_rules_android//android:rules.bzl", "android_sdk_repository")
+load("@rules_android//android:rules.bzl", "android_sdk_repository")
 
 android_sdk_repository(
     name = "androidsdk",
-    api_level = 29,
+    build_tools_version = versions.ANDROID.BUILD_TOOLS,  # versions > 30.0.3 do not have the dx.jar anymore.
```

Comment:
Might wanna link to https://github.com/bazelbuild/bazel/issues/13989 for some context